### DISCUSSION TITLE
Atom Error help

### DISCUSSION THREAD

**Discussion by shaman2527** [atom/atom#22743] [General]
![image](https://user-images.githubusercontent.com/84341975/126020824-83e9d462-d235-4ec8-932b-20b62b5fe75d.png)

hello code, every moment I open the IDE I always get that error that I can do thanks

I have the latest version every time I start Atom it gives me that error

**Comment by icecream17** [comment]
That is a very unuseful error message, so I don't know what to say.
Maybe try completely uninstalling atom, then reinstalling again?

**Reply by icecream17** [reply]
Maybe try opening in dev/safe mode: https://github.com/atom/atom/discussions/22428
Might need to search "add to path" or something. (Though if you don't understand anyways you can ask)

**Comment by the-j0k3r** [comment]
Some logs would be nice, other than just saying its broken or me too reports that dont provide any actionable information.
